sdk/src/main/scala/com/horizen/forge/AbstractForgeMessageBuilder.scala

```diff
@@ -348,7 +348,11 @@ abstract class AbstractForgeMessageBuilder[
 
   def getOmmersSize(ommers: Seq[Ommer[H]]) : Int
 
-  def getForgingStakeMerklePathInfo(nextConsensusEpochNumber: ConsensusEpochNumber, wallet: VL, history: HIS, state: MS): Seq[ForgingStakeMerklePathInfo]
+  def sortByStakeAmount(list: Seq[ForgingStakeMerklePathInfo]) : Seq[ForgingStakeMerklePathInfo] = {
```

Comment:
I think that sort operation can be done exactly in AbstractForgeMessageBuilder:
`val forgingStakeMerklePathInfoSeq: Seq[ForgingStakeMerklePathInfo] = getForgingStakeMerklePathInfo(
      nextConsensusEpochNumber, nodeView.vault, nodeView.history, nodeView.state, branchPointInfo).sort(....)`